frontend/src/components/organisms/AddingPlayerForm/AddingPlayerForm.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Button, TextField, Typography } from "@material-ui/core";
+import { Centered } from "../../atoms/Shared/Centered";
+import { VerticalSpace } from "../../atoms/Shared/VerticalSpace";
+
+function AddingPlayerForm() {
+  return (
+    <Centered>
+      <VerticalSpace height="1rem" />
+      <Typography component="h6" variant="h6">
+        Nowy zawodnik
+      </Typography>
+      <VerticalSpace height="1rem" />
+
+      <form>
+        <Centered>
+          <TextField
+            key={"name"}
+            id={`outlined-name`}
+            label={"Imię"}
+            variant={`outlined`}
+          />
+          <VerticalSpace height="1rem" />
+
+          <TextField
+            key={"surname"}
```

Comment:
Stałe nie muszą być w {}, piszesz po prostu "" jak w HTML.

Reply:
teraz już wgl usunięte :-) 
no tak inteliJ sam generuje mi te nawiasy to wale te stringi w nich :-p , ale mogę je usuwać